pulsar-broker/src/main/java/org/apache/pulsar/broker/stats/prometheus/FunctionsStatsGenerator.java

```diff
@@ -0,0 +1,78 @@
+package org.apache.pulsar.broker.stats.prometheus;
```

Comment:
I think this file is missing license header?

Reply:
good catch I will add

Reply:
I have add the missing license header